frontend/src/utils/helper.ts

```diff
@@ -101,49 +103,53 @@ export const getTrainees = (data: any, dataItems: string[]) => {
   return traineesData;
 };
 
-
-export const getApplicants = (data: ApplicantDetails[], dataItems: string[]) => {
-  return data?.map((item: any) => dataItems.map(key => item[key as keyof ApplicantDetails]));
-}
-
+export const getApplicants = (
+  data: ApplicantDetails[],
+  dataItems: string[],
+) => {
+  return data?.map((item: any) =>
+    dataItems.map((key) => item[key as keyof ApplicantDetails]),
+  );
+};
 
 /**
  * Generates a random color and text color for a chart.
  *
  * @returns {Object} An object containing the background color and text color for the chart.
- * 
+ *
  */
 
 export const getJWT = () => {
   const cookies = new Cookies();
   return cookies.get("jwt");
 };
 
-export const getRandomColorAndTextColor= () => {
-   const skyBlueHue = 200;
-   const saturation = 50;
-   const lightness = 60;
- 
-   const backgroundColor = `hsl(${skyBlueHue}, ${saturation}%, ${lightness}%, 0.4)`;
-   const textColor = "black";
- 
-   return { backgroundColor, textColor };
-};
+export const getRandomColorAndTextColor = () => {
+  const skyBlueHue = 200;
+  const saturation = 50;
+  const lightness = 60;
 
+  const backgroundColor = `hsl(${skyBlueHue}, ${saturation}%, ${lightness}%, 0.4)`;
+  const textColor = "black";
+
+  return { backgroundColor, textColor };
+};
```

Comment:
1. I think you meant `getRandomBackgroundColorAndTextColor` instead of `getRandomColorAndTextColor`.
2. I think this function will always generate the same colour because the variables that are being used will never change.